ASIO linker errors on cpal 0.17

tried running
```cargo run --example enumerate --features asio```
on the latest branch and on "0.17.0" tag

ran into:
```
$ cargo run --example enumerate --features asio
   Compiling cpal v0.17.0 (F:\CODE\cpal-0.17.0)
error: linking with `link.exe` failed: exit code: 1120
  |
  = note: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.50.35717\\bin\\HostX64\\x64\\link.exe" "/NOLOGO" "C:\\Users\\user\\AppData\\Local\\Temp\\rustcGWqfVg\\symbols.o" "<44 object files omitted>" "F:\\CODE\\cpal-0.17.0\\target\\de
indows_link-750c1e8395263f2e,libasio_sys-a3f178330438e552,libnum_traits-421cd4ba7d59e934,libanyhow-60aa99bfda26100b}.rlib" "<sysroot>\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib/{libstd-*,libpanic_unwind-*,libcfg_if-*,libwindows_targets-*,librustc_demangle-*,libstd_detect
32.lib" "ntdll.lib" "userenv.lib" "ws2_32.lib" "dbghelp.lib" "/defaultlib:msvcrt" "/NXCOMPAT" "/LIBPATH:F:\\CODE\\cpal-0.17.0\\target\\debug\\build\\asio-sys-d922dc70ba91173d\\out" "/LIBPATH:F:\\CODE\\cpal-0.17.0\\target\\debug\\build\\asio-sys-d922dc70ba91173d\\out" "/O
etc\\liballoc.natvis" "/NATVIS:<sysroot>\\lib\\rustlib\\etc\\libcore.natvis" "/NATVIS:<sysroot>\\lib\\rustlib\\etc\\libstd.natvis" "/NATVIS:C:\\Users\\user\\AppData\\Local\\Temp\\rustcGWqfVg\\enumerate-0.natvis" "/NATVIS:C:\\Users\\user\\AppData\\Local\\Temp\\rus
  = note: some arguments are omitted. use `--verbose` to show all linker arguments
  = note: libasio_sys-a3f178330438e552.rlib(66a90b18778ddb97-asiolist.o) : error LNK2019: unresolved external symbol __imp_RegCloseKey referenced in function "public: __cdecl AsioDriverList::AsioDriverList(void)" (??0AsioDriverList@@QEAA@XZ)␍
          libasio_sys-a3f178330438e552.rlib(66a90b18778ddb97-asiolist.o) : error LNK2019: unresolved external symbol __imp_RegEnumKeyA referenced in function "public: __cdecl AsioDriverList::AsioDriverList(void)" (??0AsioDriverList@@QEAA@XZ)␍
          libasio_sys-a3f178330438e552.rlib(66a90b18778ddb97-asiolist.o) : error LNK2019: unresolved external symbol __imp_RegOpenKeyA referenced in function "public: __cdecl AsioDriverList::AsioDriverList(void)" (??0AsioDriverList@@QEAA@XZ)␍
          libasio_sys-a3f178330438e552.rlib(66a90b18778ddb97-asiolist.o) : error LNK2019: unresolved external symbol __imp_RegOpenKeyExA referenced in function "long __cdecl findDrvPath(char *,char *,int)" (?findDrvPath@@YAJPEAD0H@Z)␍
          libasio_sys-a3f178330438e552.rlib(66a90b18778ddb97-asiolist.o) : error LNK2019: unresolved external symbol __imp_RegQueryValueExA referenced in function "long __cdecl findDrvPath(char *,char *,int)" (?findDrvPath@@YAJPEAD0H@Z)␍
          F:\CODE\cpal-0.17.0\target\debug\examples\enumerate.exe : fatal error LNK1120: 5 unresolved externals␍

error: could not compile `cpal` (example "enumerate") due to 1 previous error
```

doing the same on the "0.16.0" works fine.

--- a/.github/workflows/platforms.yml
+++ b/.github/workflows/platforms.yml
@@ -219,9 +219,9 @@ jobs:
       - name: Check examples (all features)
         run: cargo +${{ steps.msrv.outputs.all-features }} check --examples --all-features --workspace --verbose
 
-      - name: Check ASIO examples
+      - name: Build ASIO examples
         working-directory: asio-sys
-        run: cargo +${{ env.MSRV_WINDOWS }} check --examples --verbose --workspace
+        run: cargo +${{ env.MSRV_WINDOWS }} build --examples --verbose --workspace
 
   # macOS ARM64
   macos:

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,11 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.17.3] - 2026-02-17
+## [0.17.3] - 2026-02-18
 
 ### Changed
 
 - Reverted SemVer-breaking `DeviceBusy` error variant addition.
+
+### Fixed
+
+- **ASIO**: Fix linker errors.
 
 ## [0.17.2] - 2026-02-08 [YANKED]
 

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -80,7 +80,7 @@ windows = { version = ">=0.59, <=0.62", features = [
     "Win32_UI_Shell_PropertiesSystem",
 ] }
 audio_thread_priority = { version = "0.34", optional = true }
-asio-sys = { version = "0.2", path = "asio-sys", optional = true }
+asio-sys = { version = "0.2.6", path = "asio-sys", optional = true }
 num-traits = { version = "0.2", optional = true }
 jack = { version = "0.13", optional = true }
 

--- a/asio-sys/CHANGELOG.md
+++ b/asio-sys/CHANGELOG.md
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.6] - 2026-02-17
+
+### Fixed
+- Link `advapi32` to resolve Windows Registry API symbols.
+
 ## [0.2.5] - 2026-01-04
 
 ### Fixed
@@ -77,6 +82,7 @@ Initial release.
 - Support for MSVC toolchain on Windows
 - Basic error types: `AsioError`, `LoadDriverError`
 
+[0.2.6]: https://github.com/RustAudio/cpal/compare/asio-sys-v0.2.5...asio-sys-v0.2.6
 [0.2.5]: https://github.com/RustAudio/cpal/compare/asio-sys-v0.2.4...asio-sys-v0.2.5
 [0.2.4]: https://github.com/RustAudio/cpal/compare/asio-sys-v0.2.3...asio-sys-v0.2.4
 [0.2.3]: https://github.com/RustAudio/cpal/compare/asio-sys-v0.2.2...asio-sys-v0.2.3

--- a/asio-sys/Cargo.toml
+++ b/asio-sys/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "asio-sys"
-version = "0.2.5"
+version = "0.2.6"
 authors = ["Tom Gowan <tomrgowan@gmail.com>"]
 description = "Low-level interface and binding generation for the steinberg ASIO SDK."
 repository = "https://github.com/RustAudio/cpal/"

--- a/asio-sys/build.rs
+++ b/asio-sys/build.rs
@@ -70,6 +70,7 @@ fn main() {
     }
 
     // Print out links to needed libraries
+    println!("cargo:rustc-link-lib=dylib=advapi32");
     println!("cargo:rustc-link-lib=dylib=ole32");
     println!("cargo:rustc-link-lib=dylib=user32");
     println!("cargo:rustc-link-search={}", out_dir.display());

